src/mcp/index.ts

```diff
@@ -8,7 +8,7 @@ import {
   ListToolsResult,
 } from "@modelcontextprotocol/sdk/types.js";
 import { checkFeatureActive, mcpError } from "./util.js";
-import { SERVER_FEATURES, ServerFeature } from "./types.js";
+import { ClientConfig, SERVER_FEATURES, ServerFeature } from "./types.js";
 import { availableTools } from "./tools/index.js";
 import { ServerTool, ServerToolContext } from "./tool.js";
 import { configstore } from "../configstore.js";
```

Comment:
Not very familiar with this. Likely a dumb question

What is config store? Is it backed by a local file somewhere?

Reply:
Yeah. All device-level Firebase config is stored there. It's in `~/.config/configstore/firebase-tools.json`